### DISCUSSION TITLE
[FileService] Api portal url

### DISCUSSION THREAD

**Discussion by spampibridge** [mia-platform/community#45] [Q&A] [answered]
Hi, 

after deploying a new File Service instance, we noticed this situation on api portal:

![image](https://user-images.githubusercontent.com/114915906/193554598-a94566b0-f0fd-4d11-b174-a45ba88f4828.png)

Why the route 

`vendita/catalog-files-service/upload`

is defined under "Default" tag?

**Comment by davidebianchi** [comment]
The default tag is set if no tags are defined in the oas given by the service.

You can add a tag from the oas, and see the APIs under the correct tags. We could add the tag `files` in the files-service, what do you think?

**Reply by spampibridge** [reply]
What's oas?

**Reply by davidebianchi** [reply]
OpenAPI Specification, every service define an OpenAPI Specification (which is exposed through the `/documentation/json` path). In the spec, it is possible to write one or more tags, which are visible in API Portal as title

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, this happens because the OpenAPI Specification of the files-service plugin didn't set any tag; I've just pushed v2.6.3 which fixes this issue and uses the proper *Files Service* tag instead of the *default* one.

**Reply by davidebianchi** [reply]
We should add to the swagger-aggregator the feature to add/modify default tags exposed by the service, it should be an interesting feature!

**Reply by fredmaggiowski** [reply]
I agree
